drivers/usb/device/usb_dc_dw.c

```diff
@@ -549,6 +549,7 @@ static inline void usb_dw_int_rx_flvl_handler(void)
 	u32_t grxstsp = USB_DW->grxstsp;
 	u32_t status, xfer_size;
 	u8_t ep_idx;
+	usb_dc_ep_callback ep_cb;
```

Comment:
Oops, no one noticed that in 2161c91f0b68b04d8bafd86920a6c6cd5a9f1d67 :innocent: